SU-SWS/stanford_profile
# D8CORE-5680 Switch list landing pages to nodes with layout builder settings

# READY FOR REVIEW

# Summary
- Use nodes for /news, /events, /courses, & /publications pages
- Patched default content module to support layout builder export & import.
- Update hook to create the new node pages. This will only create a page if respective content exists.

# Need Review By (Date)
- :shrug: 

# Urgency
- low

# Steps to Test
1. checkout this branch
2. database updates or do a fresh install
3. view the respective pages to see they look the same as they did, but now they're nodes.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
